src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,240 @@
+
+This workflow relies on two existing Adobe I/O GitHub Actions published on the GitHub Marketplace:
+
+-  [`adobe/aio-cli-setup-action`](https://github.com/adobe/aio-cli-setup-action) installs and configures the [CLI](https://github.com/adobe/aio-cli) on the GitHub infrastructure. For API Mesh, this action handles authentication.
+
+- [`adobe/aio-apps-action`](https://github.com/adobe/aio-apps-action) centralizes support for a GitHub workflow to leverage application-specific commands, such as deploying via `aio api-mesh:update`.
+
+In the `.github/workflows` folder, create a `.yml` file with the following contents:
+
+```yml
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  # push:
+  #   branches: [ "main" ]
+  # pull_request:
+  #   branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+   deploy:
+    name: Deploy to Prod
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 1
+      matrix:
+        node-version: ['20']
+        os: [ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install
+        run: npm i
+      - name: Setup CLI
+        uses: adobe/aio-cli-setup-action@1.3.0
+        with:
+          os: ${{ matrix.os }}
+          version: 10.x.x
+      - name: api-mesh-plugin install
+        run: aio plugins:install @adobe/aio-cli-plugin-api-mesh
+      - name: Auth
+        uses: adobe/aio-apps-action@3.3.0
+        with:
+          os: ${{ matrix.os }}
+          command: oauth_sts
+          CLIENTID: ${{ secrets.CLIENTID_PROD }}
+          CLIENTSECRET: ${{ secrets.CLIENTSECRET_PROD }}
+          TECHNICALACCOUNTID: ${{ secrets.TECHNICALACCID_PROD }}
+          TECHNICALACCOUNTEMAIL: ${{ secrets.TECHNICALACCEMAIL_PROD }}
+          IMSORGID: ${{ secrets.IMSORGID_PROD }}
+          SCOPES: AdobeID, openid, read_organizations, additional_info.projectedProductContext, additional_info.roles, adobeio_api, read_client_secret, manage_client_secrets
+      - name: Select org
+        run: aio console:org:select ${{ secrets.ORGID_PROD }}
+      - name: Select project
+        run: aio console:project:select ${{ secrets.PROJECTID_PROD }}
+      - name: Select workspace
+        run: aio console:workspace:select ${{ secrets.WORKSPACEID_PROD }}
+      - name: 'Create env file'
+        run: |
+          touch .env
+          echo API_ENDPOINT="https://xxx.execute-api.us-west-2.amazonaws.com" >> .env
+          echo API_KEY=${{ secrets.API_KEY }} >> .env
+          cat .env
+      - name: api-mesh update
```

Comment:
Since we are mentioning just the update, we should add a note to manually create a mesh on the workspace before setting up the CI/CD. The Expectation is to have Mesh on the workspace on which CI/CD is being set up. 

cc: @naomirobichaud This is a feature gap. We need to do some work to get around this.

Reply:
@prabhuram93 this is mentioned in a note at the beginning of the page: https://github.com/AdobeDocs/graphql-mesh-gateway/pull/207/files#diff-fc6e4143b29f528c5cc247fea51e30b73af74eb31d7a71e498166362fc3316a8R27

Reply:
Sounds good.